recipes/libcorrect/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set version = "0.1" %}
+
+package:
+  name: libcorrect
+  version: {{ version }}
+
+source:
+  url: https://github.com/ryanvolz/libcorrect/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: fff88a8c4b390ca8efa8d749a11943ed1c6402058fd37446f862676b82e4a20b
+  patches:
+    - 0001-De-conflict-library-names-and-fix-paths-when-compile.patch
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('libcorrect', max_pin='x') }}
```

Comment:
I guess my logic was that this would be something I'd monitor and set the version number appropriately for this to make sense, seeing as I was the one setting the version number. I suppose that remains true, but it doesn't hurt to give more leeway in setting the version. Since we're going with "0.0.0", I propose making this `'x.x'` so at least I have the option of bumping the patch version when there is no ABI change.

Reply:
I expect you will never bump the patch version because that project has been dormant since 2018. We will lean on the side of not breaking any environments by pinning `x.x.x`. If they do update the project, then we can rebuild.